test/integration-test.py

```diff
@@ -239,7 +239,8 @@ def main():
     # traffic.
     if not args.test_case_filter:
         check_balance()
-    run_expired_authz_purger()
+    if not CONFIG_NEXT:
+        run_expired_authz_purger()
```

Comment:
See the PR description:
> Disables expired-authorization-purger integration tests if config-next is being used, a new test will be added with #4188.

Reply:
Ah, I'm guessing the problem is that no authzs get purged because they were created new style and the old style purger doesn't know about them? Not clear from the PR desc why the purger tests breaks in config-next,

Reply:
Updated description to clarify.